### PR TITLE
Added UserRowMapper. Added getListOfUsers

### PR DESCRIPTION
**Pull request by siiritann**

### CODE REVIEW

**Review by ottkaskjr — APPROVED** [review]

File: src/main/java/com/intranet/project/controller/UserController.java [hunk 1]
```diff
@@ -17,10 +16,13 @@
     private UserService userService;
 
 
-    // 1 - CREATE ACCOUNT
-    @PostMapping("/createuser")
+    @PostMapping("/create/user")
```
Comment (RohtlaS) [inline src/main/java/com/intranet/project/controller/UserController.java:19]: @RequestMapping("posting") ---- klassi ette
public class PostingController {

@PostMapping("/create") ---- meetodi ette
    public void createPosting(@RequestBody

File: src/main/java/com/intranet/project/repository/UserRowMapper.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package com.intranet.project.repository;
+
+import com.intranet.project.repository.user.UserEntity;
+import org.apache.catalina.User;
+import org.springframework.jdbc.core.RowMapper;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class UserRowMapper implements RowMapper {
```
Comment (RohtlaS) [inline src/main/java/com/intranet/project/repository/UserRowMapper.java:10]: implements RowMapper järel peaks olema <UserEntity>

Comment (RohtlaS) [reply src/main/java/com/intranet/project/repository/UserRowMapper.java:10]: <UserEntity>

File: src/main/java/com/intranet/project/controller/UserController.java (hunk 1 above)
Comment (siiritann) [reply src/main/java/com/intranet/project/controller/UserController.java:19]: done

**Review by RohtlaS — CHANGES_REQUESTED** [review]
Kommentaarid sees

File: src/main/java/com/intranet/project/repository/UserRowMapper.java (hunk 2 above)
Comment (siiritann) [reply src/main/java/com/intranet/project/repository/UserRowMapper.java:10]: done

**Pull request merged**